docs/howto/operator/gcp/dataflow.rst

```diff
@@ -0,0 +1,120 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Google Cloud Dataflow Operators
+===============================
+
+`Dataflow <https://cloud.google.com/dataflow/>`__ is a managed service for
+executing a wide variety of data processing patterns. These pipelines are created
+using the Apache Beam programming model which allows for both batch and streaming.
+
+Airflow provides operators to start/create Java and Python Dataflow jobs.
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite Tasks
+^^^^^^^^^^^^^^^^^^
+
+.. include:: _partials/prerequisite_tasks.rst
+
+Create a Java Job
+^^^^^^^^^^^^^^^^^
+
+.. _howto/operator:DataflowCreateJavaJobOperator:
+
+Using a JAR on GCS
```

Comment:
Can you include it in one section? And then describe that the file should be indicated by the parameter. This can be the local file path or the GCS file path.  Then you can create two paragraphs with two examples.

Reply:
[I think I addressed it here](https://github.com/apache/airflow/pull/8809/commits/dc48b7a99abbe26f721bb32d1e381c3b03afb946)